support-frontend/assets/helpers/productCatalog.tsx

```diff
@@ -342,3 +345,53 @@ export const productCatalogDescriptionNewBenefits: Record<
 		offers: [],
 	},
 };
+
+/**
+ * This method is to help us determine which product and rateplan to
+ * use based on a person's internationalisation ID.
+ *
+ * The reason this exists is because we have different pricing for
+ * `int` and `us` for SupporterPlus and GuardianWeekly, but they
+ * both use `USD`.
+ *
+ * As Zuora is restricted to only being able to vary on currency,
+ * we express this in the product catalog by having different products
+ * for GuardianWeekly, and different ratePlans for SupporterPlus.
+ *
+ * We are potentially going to look at Attribute based pricing in the future.
+ *
+ * @see: https://knowledgecenter.zuora.com/Zuora_Billing/Build_products_and_prices/Attribute-based_pricing/AA_Overview_of_Attribute-based_Pricing
+ * */
+export function internationaliseProductAndRatePlan(
+	supportInternationalisationId: SupportInternationalisationId,
+	productKey: ProductKey,
+	ratePlanKey: string,
+): { productKey: ProductKey; ratePlanKey: string } {
+	let productKeyToUse = productKey;
+	let ratePlanToUse = ratePlanKey;
+
+	if (productKey === 'SupporterPlus') {
+		if (supportInternationalisationId === 'int') {
+			if (ratePlanKey === 'DomesticAnnual') {
```

Comment:
Does there need to be the inverse operation of this?

Reply:
Think this is needed because f you start in region `int` then change to a country like France you get `price not found in product catalog`

Reply:
It looks like changing from France to int is missing as far as I can tell?

Reply:
Nice catch, I've added the alternative.